helm/templates/configmap.yaml

```diff
@@ -1,17 +1,18 @@
 {{- $chartName := include "task-liberator.name" . -}}
+{{- $configmapName := include "configmap.fullname" . }}
 {{- $tracingUrl := include "task-liberator.tracingUrl" . -}}
 {{- $metricsUrl := include "task-liberator.metricsUrl" . -}}
 {{- if .Values.enabled -}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}-configmap
+  name: {{ $configmapName }}
```

Comment:
Isn't it better to set the tpl to be generic instead of customize it to configmap?

Reply:
After consulting with @AnguIar and looking threw bitnami implements, we decided to keep this line of method.